.circleci/config.yml

```diff
@@ -1150,7 +1144,7 @@ jobs:
           command: bundle exec fastlane backend_integration_tests test_plan:"BackendIntegrationTests-All-CI"
           no_output_timeout: 5m
           environment:
-            SCAN_DEVICE: iPhone 14 (17.2.0)
```

Comment:
this should be 17.4.0 with xcode 15.3, and that's why integration tests don't pass atm